.github/workflows/new_contributor_pr.yml

```diff
@@ -0,0 +1,24 @@
+name: New Contributor Message
+
+on:
+  pull_request_target:
+
+jobs:
+  build:
+    name: Hello new contributor
+    runs-on: ubuntu-latest
+    steps:
+      # Pinned to v2.0
+      - uses: deborah-digges/new-pull-request-comment-action@224c179a9e23f65ec50ff3240b8716369dc415d7
+        with:
+          access-token: ${{ secrets.GITHUB_TOKEN }}
+          message: |
+            Hello ${{ github.event.pull_request.user.login }}! Thank you for your contribution 💪
+
+            As it's your first contribution be sure to check out the [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).
+            If you're fixing a ticket [from Trac](code.djangoproject.com/) make sure to set the "Has Patch" property
+            and include a link to this PR in the ticket!
+
+            If you have any design or process questions then you can ask in the [Django Development forums](https://forum.djangoproject.com/c/internals/5).
+
+            Thanks again, you rock! 🚀
```

Comment:
We often use `Welcome aboard ⛵️` — maybe just that.

Reply:
Done 👍